### PR TITLE
updating kitchen sink to beta.16

### PR DESCRIPTION
**Pull request by sumitarora**

### CODE REVIEW

**Comment by vanessayuenn** [comment]
Please update the README as well :)

**Comment by sumitarora** [comment]
done on main one this is just for example application https://github.com/rangle/augury/pull/349/files

File: example-apps/kitchen-sink-example/source/components/dynamic-controls/dynamic-controls.ts [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
     <div>
       <load-into-component></load-into-component>
       <hr/>
-      <load-next-to-component></load-next-to-component>
-      <hr/>
       <load-as-root-component></load-as-root-component>
     </div>
   `,
-  directives: [LoadIntoComponent, LoadAsRootComponent, LoadNextToComponent]
+  directives: [LoadIntoComponent, LoadAsRootComponent]
```
Comment (vanessayuenn) [inline example-apps/kitchen-sink-example/source/components/dynamic-controls/dynamic-controls.ts]: think we should keep `LoadNextToComponent` and remove `LoadIntoComponent` 😬

**Comment by vanessayuenn** [comment]
lgtm 👍 🚀

**Pull request merged**